api/v1alpha/istio-manifests/1.4/authz/allow-nothing-default-policy.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-nothing-default
+  namespace: default
+spec:
+  {}
```

Comment:
@phriscage I don't believe this one is needed as the sidecar injection is only enabled for datashare-apis from what I follow in the api docs:

https://github.com/GoogleCloudPlatform/datashare-toolkit/tree/master/api

Reply:
agreed. Feel free to remove